IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -117,6 +117,7 @@ public class ReachFive: NSObject {
                     }
                 }
             }.onFailure { error in
+                
                 let thePromise = BrightFutures.Promise<AuthToken, ReachFiveError>()
                 thePromise.failure(error)
                 _ = completion(thePromise.future)
```

Comment:
i think that i found a better solution to Improve async task, i will do it in a separated  PR.